lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -733,6 +751,12 @@ interface BindingLiquidSdk {
     [Throws=PaymentError]
     Payment? get_payment(GetPaymentRequest req);
 
+    [Throws=SdkError]
+    FetchSwapUpdatedFeesResponse fetch_swap_updated_fees(FetchSwapUpdatedFeesRequest req);
```

Comment:
In general we avoided using the term swap in the public API (we use Payment instead). 
Should we also apply this rational here? We could change this to  `FetchPaymentUpdatedFees` or `FetchPaymentProposedFees` ?